druid/src/core.rs

```diff
@@ -591,10 +602,7 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                 recurse = had_active || child_ctx.base_state.is_hot;
                 Event::Zoom(*zoom)
             }
-            Event::Timer(id) => {
-                recurse = child_ctx.base_state.request_timer;
-                Event::Timer(*id)
-            }
+            Event::Timer(token) => Event::Timer(*token),
```

Comment:
Add `recurse = false;` here because if we get this then we're already a descendant of the targeted widget.

Reply:
Good catch. Missed this important change.  :(
Thanks.